docs/pages/build/submitting-to-app-stores.md

```diff
@@ -0,0 +1,106 @@
+---
+title: Submitting builds to App Stores
+---
+
+This guide outlines how to submit your app to the Apple App Store or Google Play Store. A paid developer account is required for each platform for which you wish to submit an app. You can create an Apple Developer account on [Apple's developer site](https://developer.apple.com/account/) and a Google Play Developer account on the [Google Play Console sign-up page](https://play.google.com/apps/publish/signup/).
+
+## 1. Build a standalone app
+
+You'll need a built native app binary. You can either use the [EAS Build](introduction.md) service or do it on your own.
+
+## 2. Start the submission
+
+To submit the previously built app to the appropriate app store, run `eas submit -p ios` or `eas submit -p android` from inside of your project directory. The command will lead you step by step through the process of submitting the app. To see all possible options along with example usages, run `eas submit --help`.
+
+> Although it's possible to upload any binary to the store, each submission is assigned to an Expo project. That's why it's important to start submission from inside your project's directory - that's where your [app configuration](../workflow/configuration.md) is defined.
+
+### 2.1 Submitting your Android app to Google Play
+
+**Important:**
+
+- You have to create a [Google Service Account](https://cloud.google.com/iam/docs/creating-managing-service-accounts) and download its JSON private key.
+- After that, you'll have to create an app on [Google Play Console](https://play.google.com/apps/publish/) and upload your app manually at least once.
+
+#### Creating a Google Service Account
+
+See [expo.fyi/creating-google-service-account](https://expo.fyi/creating-google-service-account) to learn more.
+
+#### Manually uploading your app for the first time
+
+Before using `eas submit -p android` for uploading your builds, you have to upload your app manually at least once. This is a limitation of the Google Play Store API.
+
+See [expo.fyi/first-android-submission](https://expo.fyi/first-android-submission) to learn more.
+
+#### Using EAS CLI to upload the further builds of your app
+
+To upload your Android app to the Google Play Store, run `eas submit --platform android` and follow the instructions on the screen. The command will perform the following steps:
+
+1. Log in to your Expo account and ensure that your app project exists on EAS servers
+2. Prompt for the Android package name unless `expo.android.package` is set in app configuration or the `--android-package` param is provided.
```

Comment:
I think https://github.com/expo/eas-cli/pull/93 might change it :D Please sync with @wkozyra95

Reply:
for now it's still using package field, I did not add support for applicationId there in app.json. I'm just treating this field as applicationId to update native project